gwdetchar/omega/html.py

```diff
@@ -856,7 +848,7 @@ def write_qscan_page(blocks, context):
     page = markup.page()
     page.add(write_toc(blocks))
     page.h2('Channel details')
-    for block in blocks:
+    for key, block in blocks.items():
```

Comment:
shouldn't this just be `for block in blocks.values()`?

Reply:
Yes it should -- I think in the course of testing I made several changes, but forgot to go back and switch this from `items()` to `values()`. I did this in a couple of other places too, which is fixed in the latest commit.